utils/benthos_validate.py

```diff
@@ -0,0 +1,406 @@
+#!/usr/bin/env python
+
+
+"""
+Validator to ensure libais can produce Benthos's expected input
+"""
```

Comment:
Missing a period at the end of the summary sentence.

Best to go with one big doc string.  The comments below should go into the doc str.